packages/typography/__tests__/Typography.test.tsx

```diff
@@ -106,6 +107,27 @@ describe('with props.variant', () => {
   })
 })
 
+describe('with props.variant as object', () => {
+  it('have proper styles for variant with breakpoints', async () => {
+    const text = 'Variant by breakpoint test desktop'
+    render(
+      <Typography
+        variant={{
+          0: 'p4',
+          1500: 'p2',
+        }}
+      >
+        {text}
+      </Typography>,
+    )
+
+    expect(document.querySelectorAll('p').length).toBe(1)
+    expect(screen.getByText(text)).toHaveStyle(
+      'font-family: Lato,sans-serif; font-size: 1.4rem; line-height: 2rem;',
+    )
```

Comment:
For now decided to write the test which verifies only default styles (set for breakpoint 0), as react testing library isn't recognising styles from breakpoints. But style updates by breakpoints is working correctly and can be easily verified in storybook.